### ISSUE TITLE
[Bug]: [v7.0.0-beta.16] migration scripts formatting prevents clicking or copy-pasting links

### ISSUE THREAD

**Issue by thibaudcolas** [storybookjs/storybook#20444]
### Describe the bug

The "box" formatting that’s used to output results from migration scripts is making it hard to work with long links. Here is what the output looks like visually:

![storybook-formatting](https://user-images.githubusercontent.com/877585/210071111-437bb724-eaa0-4256-becd-47184e0583b9.png)

My terminal (iTerm2) normally auto-detects links so I can "cmd-click" then to open in the browser. That’s not working because of the ASCII formatting used here cutting the link text in half. This formatting also gets in the way of manually selecting the text to copy-paste it into the browser URL bar:

```
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#start-st   ││   orybook--build-storybook-binaries-removed
```

### To Reproduce

```shell
This can be reproduced on more or less any v6 project I believe so I won’t create a separate repro. Here is one I provided for #18662 if it helps: https://github.com/thibaudcolas/storybook-react-16.

Steps:

1. Get a Storybook v6 project with at least one `build-storybook` or `start-storybook` run-script.
2. Run `npx sb@next upgrade --prerelease`
3. Attempt to click the "In case this migration did not cover all of your scripts" link.
```

### System

```shell
Environment Info:

  System:
    OS: macOS 13.1

[…]

iTerm2 Build 3.4.18
zsh 5.9
```

### Additional context

As an aside I’m surprised by the use of the term "binary" in this migration script:

> Starting in Storybook 7, the `start-storybook` and `build-storybook` binaries have changed to `storybook dev` and `storybook build` respectively. In order to work with Storybook 7.0.0-beta.16, your storybook scripts have to be adjusted to use the binary. We can adjust them for you:

Those things do live in the `.bin` folder but they don’t contain binary code compiled for my machine, so I’d have expected either "executable" or even the more abstract term "command".

**Comment by IanVS** [comment]
That does seem pretty annoying.  I wonder how we could fix it...  @shilman do you have any ideas?  Maybe a link shortener?

**Comment by shilman** [comment]
I think we should get rid of the box formatting in the automigrations. It was useful to highlight key decisions when there was a lot of noise being output on the console alongside the messages. Now that the terminal noise has been removed, the boxes themselves are noisy and are causing issues (this specific issue for instance + it's also preventing us from creating CLI stories).

@MichaelArestad you might also want to weigh in here

**Comment by MichaelArestad** [comment]
@shilman You're right. I can see this being problematic. If we don't have much noise, we can remove it for now. Otherwise, we would need to work out a shorter link for it (but not a URL shortener), but I think that would still be fragile depending on terminal width and how the link wraps.

**Comment by valentinpalkovic** [comment]
As shilman and MichaelArestad agreed in this thread, the boxed formatting was removed from automigration output  -  `boxen` is no longer used anywhere in the current CLI. Closing as fixed.
